src/server/routes/fileProcessing.js

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const express = require('express');
+const Reading = require('../models/Reading');
+const moment = require('moment');
+const streamBuffers = require('stream-buffers');
+const readCSVFromString = require('../services/readCSV').readCSVFromString;
+const multer = require('multer');
+const streamToDB = require('../services/loadFromCsvStream');
+
+const router = express.Router();
+
+// The upload here ensures that the file is saved to server RAM rather than disk
+const upload = multer({ storage: multer.memoryStorage() });
+router.post('/:meter_id', upload.single('csvFile'), async (req, res) => {
```

Comment:
You need to protect this route using the authentication module (prevent non-admin users from uploading csv files). See the implementation in routes/groups.js on the groupUI branch.